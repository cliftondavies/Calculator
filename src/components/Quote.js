const Quote = () => (
  <h2>
    Quote
  </h2>
);

export default Quote;
