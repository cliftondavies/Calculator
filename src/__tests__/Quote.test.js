import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

test('renders elements on quote page', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
