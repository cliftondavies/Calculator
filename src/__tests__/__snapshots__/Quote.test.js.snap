// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders elements on quote page 1`] = `
<h2>
  Quote
</h2>
`;
